.github/workflows/stale.yml

```diff
@@ -0,0 +1,81 @@
+# See https://github.com/actions/stale
+
+name: Stale issues and pull-requests
+
+on:
+  schedule:
+    # Run once a day
+    # N.B. "should" be quoted, according to
+    # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onschedule
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    if: "github.repository == 'metoppv/improver'"
+    runs-on: ubuntu-latest
+    steps:
+        # Using actions/stale@v5
+      - uses: actions/stale@3cc123766321e9f15a6676375c154ccffb12a358
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+          # Idle number of days before marking issues/prs stale.
+          days-before-stale: 60
```

Comment:
I'd suggest increasing this time period in order to cater for existing usage of issues, such as #1522, #1737 and #1339. For example, SciTools/iris uses 500 days here.